### PR TITLE
Added ability to specify node modules to exclude

### PR DESCRIPTION
**Pull request by nkimaina**

### CODE REVIEW

**Comment by mseaton** [comment]
@nkimaina no comments on the code really, but would be interested to hear your use case that is driving this change - what do you want/expect your configuration to look like and how does this help?

**Comment by nkimaina** [comment]
@mseaton 
The motivations are two:
1. Right now, the tool tries to exclude node-modules that are not needed at run-time, e.g 'types'. It is not perfect at excluding all these modules (it was bundling some modules that were required at runtime for my case, even though I had not included them in the package.json)
2. The main use-case though is to enable the tool to be used to deploy to different environments. import-map-override config file helps with this, but it doesn't have a way to exclude modules, it just overrides already configured modules. This change adds flexibility by allowing you to exclude modules in some scenarios.

See example: https://github.com/AMPATH/ampath-poc-releasing/blob/master/package.json#L8

File: examples/openmrs-example/openmrs/frontend/import-map.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-    "@openmrs/esm-api": "/openmrs/frontend/@openmrs/esm-api@1.0.4/dist/openmrs-esm-api.js",
-    "@openmrs/esm-devtools": "/openmrs/frontend/@openmrs/esm-devtools@1.1.0/dist/openmrs-esm-devtools.js",
+    "@openmrs/esm-api": "https://localhost:8081/esm-api-overridden.js",
     "@openmrs/esm-error-handling": "/openmrs/frontend/@openmrs/esm-error-handling@1.0.1/index.js",
     "@openmrs/esm-home": "/openmrs/frontend/@openmrs/esm-home@1.0.0/dist/openmrs-esm-home.js",
     "@openmrs/esm-login": "/openmrs/frontend/@openmrs/esm-login@1.1.0/dist/openmrs-esm-login.js",
     "@openmrs/esm-primary-navigation": "/openmrs/frontend/@openmrs/esm-primary-navigation@1.0.0/dist/openmrs-esm-primary-navigation.js",
     "@openmrs/esm-root-config": "/openmrs/frontend/@openmrs/esm-root-config@1.1.1/dist/openmrs-esm-root-config.defaults.js",
     "@openmrs/esm-styleguide": "/openmrs/frontend/@openmrs/esm-styleguide@1.2.0/dist/openmrs-esm-styleguide.js",
+    "@openmrs/test-lib": "https://localhost:8082/test-lib.js",
```
Comment (brandones) [inline examples/openmrs-example/openmrs/frontend/import-map.json]: I think this is a merge error from the PR I just merged :grimacing:

**Comment by brandones** [comment]
Code looks good to me -- I'll let Mike approve per your conversation about rationale.

**Comment by brandones** [comment]
@mseaton ?

**Comment by brandones** [comment]
Let's call this good. Sorry for letting this drop, @nkimaina .

**Pull request merged**